package.json

```diff
@@ -44,6 +44,8 @@
     "eslint": "8.47.0",
     "eslint-config-next": "13.4.13",
     "framer-motion": "^10.18.0",
+    "html-react-parser": "^5.1.1",
+    "lucide-react": "^0.316.0",
```

Comment:
Why are we using the `lucide-react` library. According to [npmjs](https://www.npmjs.com/package/lucide-react) it is an icon library

Reply:
The react-feather library we have installed doesn't have the heading and all the markdown icons. .so I installed the one that has it all

Reply:
ok, for now we will leave as it is, but in future we have to find a solution for this